### DISCUSSION TITLE
Add comments to a decoupled CMS blogpost

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#12513] [Q&A]
Hello,

I'm using orchard core in another application as a decoupled CMS with razor pages, just like this one...
https://docs.orchardcore.net/en/dev/docs/guides/decoupled-cms/index.html
I want to add comments to the posts, it is possible with this module? how should I make the calling without liquid?

I used the module Commentator.OrchardCore in the project when we try to call that in page got error see below:

I not got the name part in article object:

![image](https://user-images.githubusercontent.com/5202291/192514380-47e7bc6b-d03d-421e-904a-86cf2ca6e00e.png)

When execute got error:

![image](https://user-images.githubusercontent.com/5202291/192513832-6aba6d7e-7207-4cf9-a832-9c812e08e1c5.png)

On dashboard side see:

![image](https://user-images.githubusercontent.com/5202291/192514609-f2e19008-3966-4d8d-b4e3-5cf9ba590b64.png)

please advise @hishamco @Skrypt @ns8482e 

Thank you

**Comment by hishamco** [comment]
The topic already discussed in https://github.com/OrchardCMS/OrchardCore/discussions/12491. One more thing please don't ping unless there's a reason for that, I see that you ping others over and over which is annoying

**Reply by hishamco** [reply]
For IM you could use Gitter instead

**Reply by saleemhaq** [reply]
> The topic already discussed in #12491. One more thing please don't ping unless there's a reason for that, I see that you ping others over and over which is annoying

Here I used the commentator module and try to inject that in decouple cms, but there is no luck getting error and there is u prefer the disqus module with that also no luck try to advise me I suffer from last 10 days.

**Comment by hishamco** [comment]
The topic already discussed in https://github.com/OrchardCMS/OrchardCore/discussions/12491. One more thing please don't ping unless there's a reason for that, I see that you ping others over and over which is annoying

**Reply by saleemhaq** [reply]
> The topic already discussed in #12491. One more thing please don't ping unless there's a reason for that, I see that you ping others over and over which is annoying

Here I used the commentator module and try to inject that in decouple cms, but there is no luck getting error and there is u prefer the disqus module with that also no luck try to advise me I suffer from last 10 days.

**Reply by hishamco** [reply]
Please file an issue on the module's repo

**Reply by saleemhaq** [reply]
> Please file an issue on the module's repo

Yes I file but Any idea about it, I also try my side to fix but no luck :( 

Also try to inject the disqus module but no luck any code level change need to apply for that? Please advise on I shared the whole detail:
https://github.com/OrchardCMS/OrchardCore/discussions/12491
